source/_includes/common-tasks/backups.md

```diff
@@ -48,15 +45,4 @@ When the restore is complete, Home Assistant will restart to apply the new setti
 
 Use `ha help` to get more information about the command line usage.
 
-
-### Copying your backups to another location
-
-You often need a backup in case your system has crashed. If you only store them on the crashed device, you won't be able to access them easily. We recommend that you manually copy them from `/backup` to another machine on occasion. Or even better, create an automation to handle that, or make use of one of the following add-ons:
-
-- [Google Drive Backup](https://github.com/sabeechen/hassio-google-drive-backup)
-- [Dropbox Sync](https://github.com/danielwelch/hassio-dropbox-sync)
-- [OneDrive Backup](https://github.com/lavinir/hassio-onedrive-backup)
-- [Nextcloud Backup](https://github.com/Sebclem/hassio-nextcloud-backup)
-- [Remote Backup (scp/rsync/rclone)](https://github.com/ikifar2012/remote-backup-addon)
-- [Samba backup](https://github.com/thomasmauerer/hassio-addons/tree/master/samba-backup)
-- [Syncthing](https://github.com/Poeschl/Hassio-Addons/tree/main/syncthing)
+{% include common-tasks/backups_offsite.md %}
```

Comment:
I don't think this has more use cases and therefore don't need to be extract into a snippet.